src/test/rustdoc-ui/intra-links-warning.stderr

```diff
@@ -96,8 +96,8 @@ LL | | /// [error]
    |
    = note: the link appears in this line:
            
-           [error]
-            ^^^^^
+            [error]
+             ^^^^^
```

Comment:
Is this change intentional? It looks like this is no longer stripping leading whitespace, which doesn't seem right.

Reply:
Hum... It should definitely take into account the kind of the doc fragment. Good point!